build.gradle

```diff
@@ -0,0 +1,34 @@
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath 'com.android.tools.build:gradle:1.3.0'
+  }
+}
+
+allprojects {
+  repositories {
+    mavenCentral()
+  }
+}
+
+ext {
+  minSdkVersion = 15
```

Comment:
So is there any particular reason to set the min at 15 instead of 14? We set our minimum to be 14 to be ICS and above. I realize that 14 is < 0.1% since it no longer shows up on Android Dashboards, but I cannot imagine there is a difference  between 14 and 15 that would affect this library.